api/DOMMatrix.json

```diff
@@ -5,27 +5,133 @@
+            {
+              "version_added": "5",
+              "alternative_name": "SVGMatrix"
+            },
+            {
+              "version_added": "2",
+              "alternative_name": "WebKitCSSMatrix"
+            }
+          ],
           "chrome_android": "mirror",
-          "edge": "mirror",
-          "firefox": {
-            "version_added": "33"
-          },
+          "edge": [
+            {
+              "version_added": "79"
+            },
+            {
+              "version_added": "12",
+              "alternative_name": "WebKitCSSMatrix"
+            },
+            {
+              "version_added": "12",
+              "alternative_name": "SVGMatrix"
+            },
+            {
+              "version_added": "12",
+              "version_removed": "79",
+              "alternative_name": "MSCSSMatrix"
+            }
+          ],
+          "firefox": [
+            {
+              "version_added": "33"
+            },
+            {
+              "version_added": "49",
+              "alternative_name": "WebKitCSSMatrix"
+            },
+            {
+              "version_added": "1.5",
+              "alternative_name": "SVGMatrix"
+            }
+          ],
           "firefox_android": "mirror",
-          "ie": {
-            "version_added": false
-          },
+          "ie": [
+            {
+              "version_added": "11",
+              "alternative_name": "WebKitCSSMatrix"
+            },
+            {
+              "version_added": "10",
+              "alternative_name": "MSCSSMatrix"
+            },
+            {
+              "version_added": "9",
+              "alternative_name": "SVGMatrix"
+            }
+          ],
           "oculus": "mirror",
-          "opera": "mirror",
-          "opera_android": "mirror",
-          "safari": {
-            "version_added": "11"
-          },
-          "safari_ios": "mirror",
+          "opera": [
+            {
+              "version_added": "48"
+            },
+            {
+              "version_added": "15",
+              "alternative_name": "WebKitCSSMatrix"
+            },
+            {
+              "version_added": "≤12.1",
+              "alternative_name": "SVGMatrix"
+            }
+          ],
+          "opera_android": [
+            {
+              "version_added": "48"
+            },
+            {
+              "version_added": "14",
+              "alternative_name": "WebKitCSSMatrix"
+            },
+            {
+              "version_added": "≤12.1",
+              "alternative_name": "SVGMatrix"
+            }
+          ],
+          "safari": [
+            {
+              "version_added": "11"
+            },
+            {
+              "version_added": "5",
+              "alternative_name": "SVGMatrix"
+            },
+            {
+              "version_added": "4",
+              "alternative_name": "WebKitCSSMatrix"
+            }
+          ],
+          "safari_ios": [
```

Comment:
Can we mirror here, or what's the difference if we do?

Reply:
There are some slight differences regarding Safari to Safari iOS mapping around there, so I'm skeptical about mirroring.  (If the mirroring output was the same as what's currently shown, the linter would freak out and say we should use `"mirror"`!)

Reply:
Hmm, OK. Thanks for checking!